deploy/kube-config/influxdb/chronograf-service.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    task: monitoring
+    service: chronograf
+  name: chronograf
+  namespace: kube-system
+spec:
+  # In a production setup, we recommend accessing Grafana through an external Loadbalancer
+  # or through a public IP.
+  type: LoadBalancer
+  # type: NodePort
+  ports:
+  - port: 80
+    targetPort: 10000
```

Comment:
nodePort is not required, so wouldn't it be more confusing to explicitly specify one?